addons/payment/i18n/vi.po

```diff
@@ -1389,7 +1389,7 @@ msgstr "Hướng dẫn thanh toán hoàn tiền"
 #: model:ir.model,name:payment.model_payment_token
 #: model:ir.model.fields,field_description:payment.field_payment_transaction__token_id
 msgid "Payment Token"
-msgstr "Mã thanh toán"
+msgstr "Mã Thanh toán"
```

Comment:
@kimloan2911 Token thì không nên dịch thành Mã. Nên dịch nó thành Mã bảo mật hoặc để nguyên là token. Chúng ta thấy là các ngân hàng VN giờ cũng dùng thuật ngữ Token khá nhiều nên đại đa số dân chúng sẽ dần quen với token. Nên có lẽ không cần dịch token nữa
```suggestion
msgstr "Token Thanh toán"
```
CC @leotranviindoo @royleviindoo @phamgiang2510 @Viindoo/is @Viindoo/product @Viindoo/project

Reply:
Dạ vâng anh, để em sửa lại ạ.